examples/chip-tool/gen/CHIPClientCallbacks.cpp

```diff
@@ -493,6 +493,9 @@ bool emberAfReadAttributesResponseCallback(ClusterId clusterId, uint8_t * messag
                             data[i].NodeId = emberAfGetInt64u(message, 0, messageLen);
                             message += 8;
                             CHECK_MESSAGE_LENGTH(8);
+                            data[i].Label = chip::ByteSpan(message, 32);
```

Comment:
Why is the label 32 bytes always, rather than being whatever length is required?